docs/docusaurus/docs/core/customize_expectations/_examples/define_a_source_to_target_expectation.py

```diff
@@ -0,0 +1,118 @@
+"""
+This is an example script for how to define a source-to-target Expectation.
+
+To test, run:
+pytest --docs-tests -k "docs_example_define_a_source_to_target_expectation" tests/integration/test_script_runner.py
+"""
+
+
+def set_up_context_for_example(context):
+    # Create the Data Source
+    connection_string = "sqlite:///data/yellow_tripdata.db"
+    data_source_name = "my_sql_data_source"
+    data_source = context.data_sources.add_sqlite(
+        name=data_source_name, connection_string=connection_string
+    )
+    assert data_source.name == data_source_name
+
+    # Add a Data Asset
+    asset_name = "my_data_asset"
+    database_table_name = "yellow_tripdata_sample_2019_01"
+    data_asset = data_source.add_table_asset(
+        table_name=database_table_name, name=asset_name
+    )
+    assert data_asset.name == asset_name
+
+    # Add a Batch Definition
+    batch_definition_name = "my_batch_definition"
+    batch_definition = data_asset.add_batch_definition_whole_table(
+        batch_definition_name
+    )
+    assert batch_definition.name == batch_definition_name
+
+
+# The name of this Data Source mentions Snowflake for illustrative purposes though it is not actually a Snowflake source
+def set_up_context2_for_example(context):
+    # Create the Data Source
+    connection_string2 = "sqlite:///data/yellow_tripdata.db"
+    data_source_name2 = "my_snowflake_data_source"
```

Comment:
The naming here is off. Even though it's just a name, I don't think we should name the DS after the wrong DS type. I'd probably just name them source_data_source and target_data_source